Dockerfile.deb-build

```diff
@@ -7,7 +7,12 @@ RUN echo "deb http://repos.mesosphere.io/ubuntu/ trusty main" > /etc/apt/sources
     apt-get install -y maven \
     npm \
     mesos \
+    git \
```

Comment:
why don't you just curl the archive rather than getting all the git stuff with it (and have to install git)